docs/src/user/starting-linuxcnc.adoc

```diff
@@ -35,19 +35,19 @@ Options:
     -k: Continue in the presence of errors in .hal files
     -t "tpmodulename [parameters]"
             specify custom trajectory_planning_module
-            overrides optional ini setting [TRAJ]TPMOD
+            overrides optional INI setting [TRAJ]TPMOD
     -m "homemodulename [parameters]"
             specify custom homing_module
-            overrides optional ini setting [EMCMOT]HOMEMOD
+            overrides optional INI setting [EMCMOT]HOMEMOD
     -H "dirname": search dirname for Halfiles before searching
-                  ini directory and system library:
+                  INI directory and system library:
```

Comment:
This is a 1:1 printout of the command `linuxcnc -h`, so I would not change this.

Reply:
No, please accept and we also fix the `linuxcnc -h` accordingly.

Reply:
There are quite a lot of "ini"s in the source tree. I already started and will prepare a separate PR for that - am not sure that we want to change them all.

Reply:
Yeah but we shouldn't merge things that we know that they need to be changed later...